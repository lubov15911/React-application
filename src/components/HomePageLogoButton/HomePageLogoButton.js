import React from 'react';

import './HomePageLogoButton.scss'

const HomePageLogoButton = (props) => {
    return (<button className="home-page-btn" onClick={props.handleClick}><b>netflix</b>roulette</button>);
};

export default HomePageLogoButton;
