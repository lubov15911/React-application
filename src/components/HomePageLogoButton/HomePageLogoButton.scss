.home-page-btn {
  color: #f65261;
}
