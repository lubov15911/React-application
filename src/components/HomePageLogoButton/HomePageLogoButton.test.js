import React from 'react';
import { shallow } from 'enzyme';

import HomePageLogoButton from './HomePageLogoButton';

describe('HomePageLogoButton', () => {
    it('should render correctly', () => {
        let routeToHomePage = () => {};
        const component = shallow(<HomePageLogoButton handleClick={routeToHomePage} />);

        expect(component).toMatchSnapshot();
    });
});
