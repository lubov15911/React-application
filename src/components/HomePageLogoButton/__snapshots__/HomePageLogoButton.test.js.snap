// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePageLogoButton should render correctly with no props 1`] = `
<button
  className="home-page-btn"
  onClick={[Function]}
  type="button"
>
  <b>
    netflix
  </b>
  roulette
</button>
`;
